main/src/ca/uwaterloo/flix/language/ast/TypeConstructor.scala

```diff
@@ -408,4 +408,9 @@ object TypeConstructor {
     */
   case class Error(kind: Kind) extends TypeConstructor
 
+  @IntroducedBy(MonoDefs.getClass)
+  case object AnyType extends TypeConstructor {
```

Comment:
You have just  thrown it in here, but I think it belongs next to Void and Unit. I would probably put it between those two.